crime-commons-mod-schemas/src/main/resources/schemas/ioj/apiPerformIojAppealResponse.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "apiPerformIojAppealResponse.json",
+  "type": "object",
+  "title": "Perform IoJ Appeal response",
+  "description": "Data contract for the response to create or update an IoJ Appeal",
+  "properties": {
+    "appealId": {
+      "type": "string",
+      "description": "The UUID of the appeal"
+    },
+    "legacyAppealId": {
+      "type": "integer",
+      "description": "The ID of the legacy appeal (defined in MAAT DB)"
+    }
+  },
```

Comment:
Are we sure we want to return just the IDs here and not the object that has been created/updated so that orchestration or MAAT can reference it?

Reply:
Yes, see the schema design page on Confluence for the reasons why.

Reply:
Ah yeh read through the explanation on the Confluence page and that makes sense 👍